tsuru/main.go

```diff
@@ -228,21 +222,14 @@ func recoverCmdPanicExitError() {
 
 func main() {
 	defer recoverCmdPanicExitError()
+	defer config.SaveChangesWithTimeout()
 
-	if inDockerMachineDriverMode() {
-		err := dockermachine.RunDriver(os.Getenv(localbinary.PluginEnvDriverName))
-		if err != nil {
-			log.Fatalf("Error running driver: %s", err)
-		}
-	} else {
-		defer config.SaveChangesWithTimeout()
+	checkVerResult := selfupdater.CheckLatestVersionBackground(version)
+	defer selfupdater.VerifyLatestVersion(checkVerResult)
 
-		checkVerResult := selfupdater.CheckLatestVersionBackground(version)
-		defer selfupdater.VerifyLatestVersion(checkVerResult)
+	localbinary.CurrentBinaryIsDockerMachine = true
```

Comment:
@tcarreira localbinary is not necessary

Reply:
I'm not sure what it does, I did not change it. 
You think I should I remove that line?

Reply:
removed ;)